src/filters/cuckoofilter.rs

```diff
@@ -481,6 +533,35 @@ where
             self.table.set(pos as u64, data);
         }
     }
+
+    /// Clear and load filter table with individual filter table elements
+    /// and existing element count.
+    pub fn load_table<I: IntoIterator<Item = u64>>(&mut self, table: I, n_elements: usize) {
+        self.clear();
+        for (i, value) in table.into_iter().enumerate() {
```

Comment:
This should probably validate that the iterator has the correct length because this depends on the filter parameters.